### ISSUE TITLE
investigate newer wifi driver for Nitrogen8M Mini SBC that supports con_mode option

### ISSUE THREAD

**Issue by floion** [balena-os/balena-fsl-arm#100]
The newer version of the wifi driver in the source branch `boundary-CNSS.LEA.NRT_2.0` reportedly supports con_mode. See https://github.com/boundarydevices/qcacld-2.0 for details
This newer driver also needs the appropriate firmware.

**Comment by acostach** [comment]
Wifi appears to work normally after switching driver and firmware to the requested branches and versions. Another wifi direct interface gets created  with these versions, therefore we set it to 'unmanaged' so that NM does not pick it to associate with wifi networks.

To switch mode, from container it is possible to do:
```
rmmod wifi
modprobe wlan con_mode=4
```

after this we can see:
```
root@a0e232b:~# iw dev
phy#1
        Interface wlan0
                ifindex 11
                wdev 0x100000001
                addr 3c:e1:a1:e0:09:f0
                type monitor
                txpower 0.00 dBm

```
[PR](https://github.com/balena-os/balena-fsl-arm/pull/101) that addresses this has been merged.
